axelor-production/src/main/java/com/axelor/apps/production/service/manuforder/ManufOrderStockMoveService.java

```diff
@@ -142,6 +143,27 @@ public StockMove _createToConsumeStockMove(ManufOrder manufOrder, Company compan
             ? company.getStockConfig().getOutsourcingReceiptStockLocation()
             : getDefaultStockLocation(manufOrder, company, STOCK_LOCATION_IN);
 
+    if (fromStockLocation == null) {
```

Comment:
I'm not sure about adding these check in this method. Depends on wether we allow source stock location and destination stock location to be null or not.